rust/main/chains/hyperlane-ethereum/src/ism/interchain_security_module.rs

```diff
@@ -94,6 +95,14 @@ where
     }
 }
 
+// The address 0x69BE704F62F7CbC1a30E35E0153D89e2b0A6Aa55 as a byte array.
+// This address was randomly generated in order to estimate gas better than
```

Comment:
> to estimate gas better

is it better because it doesn't mislead the reader that any pre-existing balance / usage is required on the address?

Reply:
I believe it's because of some compression gains if an address is low entropy, I'll add it to the comment